src/CoreBundle/private/library/classes/dbobjects/TCMSTreeNode.class.php

```diff
@@ -704,7 +704,7 @@ public function TriggerUpdateOfPathCache($bIgnoreWorkflow = false)
         $oTableConf = $this->GetTableConf();
         /** @var $oChildEditor TCMSTableEditorManager */
         $oChildEditor = new TCMSTableEditorManager();
-        $oChildEditor->Init($oTableConf->id, $this->id);
+        $oChildEditor->Init($oTableConf->id, $this->id, $this->iLanguageId);
```

Comment:
The type hint in the docblock claims `$this->iLanguageId` can only be an int. However it has a default value of `null` and is therefore nullable. `$oChildEditor->Init` on the other hand claims to need the parameter `$sLanguageId` to be a string but also has a default value of `null`.

I corrected the hints in the docblocks, but since the hints are only present in the doc blocks, I think that PHP will type juggle here automatically and correctly. Regardless, I have now made what is happening more explicit by casting to string  myself if `$this->iLanguageId` is not `null`.

I also took the liberty to correct the in-line docblock a few lines before. It should be:

```
/** @var <type> $<varName> */
```

See: https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/var.html#var